src/main/java/org/wso2/carbon/connector/pcml/PCMLPool.java

```diff
@@ -0,0 +1,64 @@
+package org.wso2.carbon.connector.pcml;
```

Comment:
Shall we add the license headers? Please add them to `PCMLPoolMBean` class as well

Reply:
done